trendico/views.py

```diff
@@ -175,3 +175,45 @@ def post(self, request, cart_item_id):
     def handle_no_permission(self):
         messages.error(self.request, "You need to log in to access this page.")
         return redirect('login')
+
+
+class AddToWishlistView(LoginRequiredMixin, View):
```

Comment:
we can make wishlist listing here as well and rather than using `AddToWishlistView`
it can be `WishlistView` and handle listing and creation

Reply:
you mean listing as fetching the list of wishlist by get method?

Reply:
yes